Annotations/AddCustomStampInCustomizedButton/App.config

```diff
@@ -1,6 +1,6 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7" />
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8"/>
```

Comment:
Check whether this changes is done for all the config files

Reply:
Check all the config file for needed changes completed